common/utils/src/main/resources/error/error-classes.json

```diff
@@ -860,6 +860,16 @@
       "Exceeds char/varchar type length limitation: <limit>."
     ]
   },
+  "EXPECT_TABLE_NOT_VIEW" : {
+    "message" : [
+      "The view <viewName> does not support <operation>, Please use ALTER VIEW instead."
```

Comment:
I think `Please use ALTER VIEW instead` only applies to certain commands?

Reply:
We can add sub error classes
```
NO_ALTERNATIVE: ...
USE_ALTER_VIEW: ...
```

Reply:
We can add more sub error clasess when some commands start to have alternative commands.

Reply:
Okay, I'll do it.